README.md

```diff
@@ -51,161 +43,110 @@ source and public models in popular formats such as TensorFlow, ONNX, PaddlePadd
 
-More Information is available at https://docs.openvino.ai/latest/openvino_docs_telemetry_information.html.
+### Tutorials
 
-## Documentation
+Explore a variety of tutorials in the [OpenVINO Notebooks Repository](https://github.com/openvinotoolkit/openvino_notebooks)📚.
 
-### User documentation
+Check out these notebooks that show how to optimize and deploy popular models (no installation required):
+- [Create an LLM-powered Chatbot using OpenVINO](https://github.com/openvinotoolkit/openvino_notebooks/blob/main/notebooks/254-llm-chatbot/254-llm-chatbot.ipynb)
+- [YOLOv8 Optimization](https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks/230-yolov8-optimization)
+- [Text-to-Image Generation](https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks/235-controlnet-stable-diffusion)
 
-The latest documentation for OpenVINO™ Toolkit is available [here](https://docs.openvino.ai/). This documentation contains detailed information about all OpenVINO components and provides all the important information you may need to create an application based on binary OpenVINO distribution or own OpenVINO version without source code modification.
+### Products Powered by OpenVINO
 
-### Developer documentation
+#### OpenVINO Ecosystem
 
-[Developer documentation](./docs/dev/index.md) contains information about architectural decisions which are applied inside the OpenVINO components. This documentation has all necessary information which could be needed in order to contribute to OpenVINO.
+-	[🤗Optimum Intel](https://github.com/huggingface/optimum-intel) -  a simple interface to optimize Transformers and Diffusers models.
+-   [Neural Network Compression Framework (NNCF)](https://github.com/openvinotoolkit/nncf) - advanced model optimization techniques including quantization, filter pruning, binarization and sparsity.
+-   [OpenVINO™ Model Server (OVMS)](https://github.com/openvinotoolkit/model_server) - a scalable, high-performance solution for serving models optimized for Intel architectures.
+-   [Software Development Kit (SDK) for the Intel® Geti™](https://github.com/openvinotoolkit/geti-sdk) - an interactive video and image annotation tool for computer vision use cases.
 
-## Tutorials
+#### External OpenVINO-Powered Products
 
-The list of OpenVINO tutorials:
+[GIMP AI plugins](https://github.com/intel/openvino-ai-plugins-gimp), [Frigate](https://github.com/blakeblackshear/frigate), [whisper.cpp](https://github.com/ggerganov/whisper.cpp) and many other products showing OpenVINO's flexibility across different applications.
 
-- [Jupyter notebooks](https://github.com/openvinotoolkit/openvino_notebooks)
+Check out [Awesome OpenVINO](https://github.com/openvinotoolkit/awesome-openvino) repository to see all the community-made projects using OpenVINO!
 
-## Products which use OpenVINO
+<details>
+  <summary>More products ### this will be removed when we decide on the products to highlight </summary>
```

Comment:
@moslex can you help with this section?

Reply:
let me block merge until decision on this section will be made, ping me when discussion will be solved, please

Reply:
I would recommend to have this link only instead of listing products which use OpenVINO:
A curated list of OpenVINO based AI projects: https://github.com/openvinotoolkit/awesome-openvino